### ISSUE TITLE
PHP-7.4: PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in /vendor/phing/phing/classes/phing/system/io/UnixFileSystem.php on line 97

### ISSUE THREAD

**Issue by mfn** [phingofficial/phing#1210]
When I run phing 2.16.1  for our deployment,  always get this:
```
PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in vendor/phing/phing/classes/phing/system/io/UnixFileSystem.php on line 97
PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in vendor/phing/phing/classes/phing/system/io/UnixFileSystem.php on line 126
PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in vendor/phing/phing/classes/phing/system/io/UnixFileSystem.php on line 139
PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in vendor/phing/phing/classes/phing/system/io/UnixFileSystem.php on line 177
PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in vendor/phing/phing/classes/phing/system/io/UnixFileSystem.php on line 197
PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in vendor/phing/phing/classes/phing/system/io/UnixFileSystem.php on line 258
```

It's not "breaking", still works but emits this message.

**Comment by mrook** [comment]
Related to #1205.

**Comment by mrook** [comment]
Fixed in #1215
